src/Abp/Text/FormattedStringValueExtracter.cs

```diff
@@ -85,6 +82,24 @@ public ExtractionResult Extract(string str, string format, bool ignoreCase = fal
             return result;
         }
 
+        private List<FormatStringToken> TokenizeFormat(string originalFormat, char splitformatCharacter)
```

Comment:
You can add default value to splitformatCharacter. Or as better, why don't you define this as nullable (also for Extract method too)?